cpp-ch/CMakeLists.txt

```diff
@@ -40,7 +40,7 @@ if("${CH_SOURCE_DIR}" STREQUAL "${CMAKE_SOURCE_DIR}/ClickHouse")
       COMMAND git reset --hard ${CH_COMMIT}
       WORKING_DIRECTORY ${CH_SOURCE_DIR} COMMAND_ERROR_IS_FATAL ANY)
     execute_process(
-      COMMAND git submodule update --init --force --depth 1
+      COMMAND git submodule update --init --force --depth 1 --jobs 32
```

Comment:
Try `--single-branch` ?

Reply:
thanks, it can significantly reduce the time and disk space usage of the clone operation.